xikolo-ios/ViewControllers/Courses/VideoViewController.swift

```diff
@@ -16,13 +16,17 @@ class VideoViewController : UIViewController {
     @IBOutlet weak var titleLabel: UILabel!
     @IBOutlet weak var descriptionTextView: UITextView!
     @IBOutlet weak var openSlidesButton: UIButton!
-    @IBOutlet weak var previousItemButton: UIButton!
-    @IBOutlet weak var nextItemButton: UIButton!
     @IBOutlet weak var summaryView: UITextView!
 
     var courseItem: CourseItem!
     var video: Video?
 
+    @IBAction func openSlides(sender: UIButton) {
+        if video != nil {
+            performSegueWithIdentifier("ShowSlides", sender: video)
+        }
```

Comment:
No need to check for `nil` as this button will not be visible if it's `nil` anyway.